### ISSUE TITLE
ability to manage several repos on one screen

### ISSUE THREAD

**Issue by coolaj86** [tjanczuk/gissues#3]
I want to use this with one of my organizations and look at several of the repos at once.

**Comment by tjanczuk** [comment]
I was planning to keep the functionality of the app simple and so I would be inclined to wait until this request has more demand behing it. 

However, I just made the source of the app public, feel free to fork and adjust it as you see fit.

**Comment by coolaj86** [comment]
I was just about to ask for the source.

I figured it was a pretty simple endeavor so I started on my own and I was hoping to be able to nab some code from yours.

**Comment by tjanczuk** [comment]
Sure, help yourself.

**Comment by cognifloyd** [comment]
Yes please. I have a couple of tightly coupled repos that I would like to manage together (or even, move issues from one repo to the other when I start organizing the issues by component).
